### PR TITLE
Update issue template

### PR DESCRIPTION
**Pull request by Hlavtox**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add information about corp modules.
| Type?             | 
| Category?         | PM
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | I
| UI Tests          | 
| Fixed issue or discussion?     | Fixes spamming the repository with billion reports about badly written company modules that external contributors don't have access to.
| Related PRs       | 
| Sponsor company   |

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1_bug_report.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ body:
       value: |
          ### ❗️ Read this before submitting your bug report:
          - **Write in English.** Reports in all other languages will be closed.
+         - **Do not open issues about following modules**  - klaviyopsautomation, mbeshipping, ps_accounts, ps_edition_basic, ps_eventbus, ps_facebook, ps_checkout, ps_mbo, ps_metrics, psxmarketingwithgoogle. Contact prestashop.com support instead.
```
Comment (PululuK) [inline .github/ISSUE_TEMPLATE/1_bug_report.yml:10]: I suggest to add a generic message about addons modules

Comment (Hlavtox) [reply .github/ISSUE_TEMPLATE/1_bug_report.yml:10]: People don't know what are addon modules. They download the bloated distro from prestashop.com thinking it's a clean installation.

**Review by matks — CHANGES_REQUESTED** [review]
I am against this change because it's PrestaShop SA related.

If tomorrow, I don't know, Microsoft creates its own PrestaShop distribution with a ps_microsoft module and it's available on microsoft.com, will we add them to the list? And then with Google? And then with...

The real message should be "Please only report issues about the PrestaShop project: any project/product that does not belong to the PrestaShop GitHub organization is out of scope"

But I'm quite sure the people who report the bugs you want to avoid will not understand the above message

I believe you won't be able to filter out these issues effectively. No matter what you write, they will be opened, and we'll have to answer to them.

**Comment by Hlavtox** [comment]
@matks Yes, but Microsoft will not confuse the users by making them think that they are downloading the software from the official website.

You put Prestashop to google and first link is prestashop.com edition with this bloatware.

**Pull request closed without merging**